packages/fast-css-editor-react/src/position/position.tsx

```diff
@@ -1,50 +1,43 @@
 import * as React from "react";
-import { ManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
-import manageJss, { ManagedJSSProps } from "@microsoft/fast-jss-manager-react";
-import style, { CSSPositionClassNameContract } from "./position.style";
-
-export enum PositionValue {
-    static = "static",
-    absolute = "absolute",
-}
-
-export enum Location {
-    top = "top",
-    left = "left",
-    right = "right",
-    bottom = "bottom",
-}
-
-export interface LocationsMappedToClassNames {
-    location: Location;
-    className: string;
-}
-
-export interface CSSPositionProps {
-    position?: PositionValue;
-    left?: string;
-    right?: string;
-    top?: string;
-    bottom?: string;
-    onChange?: (positionValues: any) => void;
-}
-
-class CSSPosition extends React.Component<
-    CSSPositionProps & ManagedClasses<CSSPositionClassNameContract>,
+import Foundation, {
+    FoundationProps,
+    HandledProps,
+} from "@microsoft/fast-components-foundation-react";
+import {
+    CSSPositionHandledProps,
+    CSSPositionUnhandledProps,
+    CSSPositionValues,
+    Location,
+    LocationsMappedToClassNames,
+    PositionValue,
+} from "./position.props";
+
+export default class CSSPosition extends Foundation<
+    CSSPositionHandledProps,
+    CSSPositionUnhandledProps,
     {}
 > {
-    private positionKey: string = "position";
+    public static displayName: string = "CSSPosition";
+
+    protected handledProps: HandledProps<CSSPositionHandledProps> = {
+        position: void 0,
+        top: void 0,
+        bottom: void 0,
+        left: void 0,
+        right: void 0,
+        onPositionUpdate: void 0,
+        managedClasses: void 0,
+    };
 
-    public render(): JSX.Element {
+    public render(): React.ReactElement<HTMLDivElement> {
```

Comment:
We were running into some issues with returning React.ReactElement types - react.d.ts types this as `React.ReactNode` so we should probably align